files/common/smart_contracts/scripts/deploy.js

```diff
@@ -0,0 +1,76 @@
+const path = require('path');
+const fs = require('fs-extra');
+const Web3 = require('web3');
+
+// WARNING: the keys here are demo purposes ONLY. Please use a tool like Orchestrate or EthSigner for production, rather than hard coding private keys
+const member1AccountAddress = "0xf0e2db6c8dc6c681bb5d6ad121a107f300e9b2b5"
```

Comment:
Rather than redefining here, can we set these to use `keys.js` as an import similar to https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/goquorum/smart_contracts/scripts/private_tx.js#L7 . The entire file also appears to be the same as https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/goquorum/smart_contracts/scripts/private_tx.js?